apps/console/src/features/applications/pages/applications.tsx

```diff
@@ -426,9 +406,12 @@ const ApplicationsPage: FunctionComponent<ApplicationsPageInterface> = (
                                     ".placeholder")
                                 }
                                 placeholder={ t("console:develop.features.applications.advancedSearch.placeholder") }
+                                style={ { minWidth: "500px" } }
```

Comment:
Don't use in-line styling. Let's use a CSS class instead. Check in other places as well.

Reply:
We have used in-line styling in some other places also in the codebase when we have only single style for an element. A element style overriding file is dedicated with classes only If there are multiple styles,